### PR TITLE
Revert the cgo PRs - they appear to cause zedrouter memory leak and OOM crash

### PR DESCRIPTION
**Pull request by eriknordmark**

### CODE REVIEW

File: pkg/pillar/go.mod [hunk 1]
```diff
@@ -13,18 +13,18 @@ require (
...
 	github.com/fsnotify/fsnotify v1.4.7
-	github.com/go-logfmt/logfmt v0.4.0 // indirect
 	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/gogo/googleapis v1.3.2 // indirect
 	github.com/golang/protobuf v1.3.2
 	github.com/google/go-cmp v0.3.0
 	github.com/google/go-containerregistry v0.0.0-20200123184029-53ce695e4179
 	github.com/google/go-tpm v0.1.1
-	github.com/google/gopacket v1.1.17
+	github.com/google/gopacket v1.1.16
```
Comment (deitch) [inline pkg/pillar/go.mod]: Why the drop in gopacket version?

File: pkg/pillar/cmd/zedrouter/flowstats.go [hunk 2]
```diff
@@ -14,16 +14,15 @@ import (
...
 	"github.com/google/gopacket"
 	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
 	"github.com/lf-edge/eve/pkg/pillar/types"
-	pcap "github.com/packetcap/go-pcap"
-	uuid "github.com/satori/go.uuid"
+	"github.com/satori/go.uuid"
 	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"syscall"
```
Comment (deitch) [inline pkg/pillar/cmd/zedrouter/flowstats.go]: Can we keep the gofmt cleanup of the imports?

File: pkg/pillar/cmd/zedrouter/flowstats.go [hunk 3]
```diff
@@ -624,9 +624,9 @@ func DNSMonitor(bn string, bnNum int, ctx *zedrouterContext, status *types.Netwo
 		switched = true
 		filter = "udp and (port 53 or port 67)"
 	}
-	log.Infof("(FlowStats) DNS Monitor on %s(bridge-num %d) switched=%v, filter=%s", bn, bnNum, switched, filter)
+	log.Infof("(FlowStats) DNS Monitor on %s(bridge-num %d) swithced=%v, filter=%s", bn, bnNum, switched, filter)
```
Comment (deitch) [inline pkg/pillar/cmd/zedrouter/flowstats.go]: We should fix this typo anyways.

File: pkg/pillar/cmd/zedrouter/flowstats.go [hunk 4]
```diff
@@ -611,7 +610,8 @@ func DNSMonitor(bn string, bnNum int, ctx *zedrouterContext, status *types.Netwo
 		snapshotLen int32 = 1280             // draft-madi-dnsop-udp4dns-00
 		promiscuous       = true             // mainly for switched network
 		timeout           = 10 * time.Second // collect enough packets in 10sec before processing
-		filter            = "udp and port 53"
+		handle      *pcap.Handle
+		filter      = "udp and port 53"
```
Comment (deitch) [inline pkg/pillar/cmd/zedrouter/flowstats.go]: Let's keep these gofmt fixes?

**Comment by eriknordmark** [comment]
> Overall, sadly, this is necessary. I had a question around google/gopacket version, and we should keep some of the neutral gofmt fixes.

I view a revert as a mechanical process of a complete PR or in exceptional cases of one commit from a PR. That makes it easier to re-apply the commits later once we know the fix.

**Comment by deitch** [comment]
Will do. @Eriknordmark, did you get size info for before and after to compare?

**Pull request merged**